packages/crayons-extended/custom-objects/src/components/form-builder/form-builder.tsx

```diff
@@ -1051,33 +1128,13 @@ export class FormBuilder {
               >
                 {fieldTypeElements}
               </fw-drag-container>
-              {this.role === 'trial' && (
-                <div class={`${strBaseClassName}-left-panel-list-disabled-div`}>
-                  <fw-icon name='lock' size='30'></fw-icon>
-                  <label
-                    class={`${strBaseClassName}-left-panel-list-disabled-header`}
-                  >
-                    {i18nText(
-                      objProductPresetConfig?.freePlanFieldAddDisabledHeader
-                    )}
-                  </label>
-                  <label
-                    class={`${strBaseClassName}-left-panel-list-disabled-message`}
-                  >
-                    {i18nText(
-                      objProductPresetConfig?.freePlanFieldAddDisabledMessage
-                    )}
-                  </label>
-                  <fw-button
-                    color='primary'
-                    onFwClick={this.explorePlanHandler}
-                    class={`${strBaseClassName}-left-panel-list-disabled-button`}
-                  >
-                    {i18nText(
-                      objProductPresetConfig?.freePlanFieldAddDisabledButton
-                    )}
-                  </fw-button>
-                </div>
+              {this.renderDisableFieldCreateByRole(
+                objProductPresetConfig,
+                strBaseClassName
+              )}
+              {this.renderDisableFieldCreateByPermission(
```

Comment:
if role is trial, we are seeing two overlays. One overlay for the role and on top of it, we are seeing the overlay for creation. Is this expected? @sangeet-fw

Reply:
Yes, One is for role and the other one is for not having create permission - The UI, message, button for explore is different

Reply:
Thanks for picking this Rihan - Fixed it